src/components/common/Dropdown.jsx

```diff
@@ -93,7 +93,11 @@ const Dropdown = ({
         <button
```

Comment:
Can we use our Button component here?

Reply:
This isn't really used as a button in the Dropdown. Bulma basically turns the `<button>` in the dropdown-trigger div into a `<select>`. I left it as default button tag to avoid applying unintentional styles later. 

I added `className` to Dropdown props. We can use this to customize dropdown components if needed.